### DISCUSSION TITLE
OpenWeatherMap One Call Version 2.5 will be deprecated in June

### DISCUSSION THREAD

**Discussion by cloneofghosts** [breezy-weather/breezy-weather#933] [General]
I'd create an issue for this but I wasn't sure which issue template to use as none of them really match.

OpenWeatherMap is deprecating Version 2.5 of their One Call API in favor of Version 3.0. Visiting https://openweathermap.org/api/one-call-api shows this message:

> One Call API 2.5 will be deprecated in June 2024. Please read the [detailed guide](https://openweathermap.org/one-call-transfer) for transitioning to the more advanced [One Call API 3.0](https://openweathermap.org/api/one-call-3) which includes several useful upgrades like 40+ years of history archive, daily aggregation and human readable daily summary in comparison with One Call API 2.5 version. Read more about One Call API 3.0 in the product documentation [page](https://openweathermap.org/api/one-call-3) and [FAQ](https://openweathermap.org/faq#onecall).

You haven't been able to make a API key for One Call 2.5 for a while now since when you try to signup you are doing so for Version 3.0. Looks like the only way to use their One Call API is to enable billing on your account.

**Comment by papjul** [comment]
Thanks for the notice!

I can schedule the removal of the bundled API key once we have a more precise date.

Since I don't want to give out my billing info, I will also need to remove the source completely as I won't be able to maintain the source without an API key.
